spec/grammar-spec.coffee

```diff
@@ -95,27 +111,51 @@ describe "less grammar", ->
     expect(tokens[0]).toEqual value: "#", scopes: ['source.css.less', 'meta.selector.css', 'entity.other.attribute-name.id', 'punctuation.definition.entity.css']
     expect(tokens[1]).toEqual value: "abc-123", scopes: ['source.css.less', 'meta.selector.css', 'entity.other.attribute-name.id']
 
+  it "parses pseudo classes", ->
+    {tokens} = grammar.tokenizeLine(".foo:hover { span:last-of-type { font-weight: bold; } }")
+    expect(tokens[0]).toEqual value: ".", scopes: ['source.css.less', 'entity.other.attribute-name.class.css', 'punctuation.definition.entity.css']
+    expect(tokens[1]).toEqual value: "foo", scopes: ['source.css.less', 'entity.other.attribute-name.class.css']
+    expect(tokens[2]).toEqual value: ":", scopes: ['source.css.less', 'entity.other.attribute-name.pseudo-class.css', 'punctuation.definition.entity.css']
+    expect(tokens[3]).toEqual value: "hover", scopes: ['source.css.less', 'entity.other.attribute-name.pseudo-class.css' ]
+    expect(tokens[4]).toEqual value: " ", scopes: ['source.css.less']
```

Comment:
Well, sometimes it's useful to check the scoped of the whitespace as well. In this case it might be a bit much, but it does guard against accidentally including whitespace where a lookahead would be better. So, I'm kind of in favour of keeping this and testing too much, over hunting down the spec for things we might not need to test.